.github/workflows/pull-request.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       - name: Build and setup everything with bazel
         id: forkliftci
-        uses: kubev2v/forkliftci@v1.0
+        uses: kubev2v/forkliftci@main
```

Comment:
since we use versioning now, shouldn't it be v<something> and not main?

Reply:
yes, but i want to to test the PR against the latest CI code (since  the tests depends on it)

Reply:
btw, I tried to make it configurable by using a variable that will appear in both cases but [github doesn't support it](https://github.com/orgs/community/discussions/25246)

Reply:
should it change now?